homeassistant/components/bmw_connected_drive/coordinator.py

```diff
@@ -79,8 +78,8 @@ async def _async_update_data(self) -> None:
                 self.account.refresh_token,
             )
 
-        # Reset scan interval after successful update
-        self.update_interval = timedelta(seconds=DEFAULT_SCAN_INTERVAL_SECONDS)
+        if self.last_update_success is False:
+            _LOGGER.info("Reconnected to BMW API")
```

Comment:
Coordinator already have logging so why this additional logging and the one above?

Reply:
You're right, I removed it.
Suppose I added it because the developer manual states "should log once when reconnected" somewhere.